packages/shared/src/components/GoToDevCardButton.tsx

```diff
@@ -1,30 +1,37 @@
 import React, { ReactElement, ReactNode } from 'react';
-import classNames from 'classnames';
 import useGoToDevCardButton from '../hooks/useGoToDevCardButton';
 import { Button } from './buttons/Button';
+import { SimpleTooltip } from './tooltips/SimpleTooltip';
 
 export type GoToDevCardButtonProps = {
   origin: string;
   children?: ReactNode;
   className?: string;
+  isLocked?: boolean;
 };
 
 export default function GoToDevCardButton({
   origin,
   children,
   className,
+  isLocked,
 }: GoToDevCardButtonProps): ReactElement {
   const [onGenerateDevCardClick] = useGoToDevCardButton(origin);
 
   return (
-    <Button
-      className={classNames('btn-primary', className)}
-      tag="a"
-      buttonSize="small"
-      href={`${process.env.NEXT_PUBLIC_WEBAPP_URL}devcard`}
-      onClick={onGenerateDevCardClick}
-    >
-      {children}
-    </Button>
+    <SimpleTooltip content={isLocked && 'Sign up to Unlock'}>
```

Comment:
Does it now not render when you hover it in non locked mode?

Reply:
Yes it does now 🚀 

![Screen Shot 2022-02-04 at 9 40 58 PM](https://user-images.githubusercontent.com/13744167/152538621-109dbbd8-1310-40a3-b546-bf7b54f814a3.png)